sql/parser/all_keywords.awk

```diff
@@ -6,7 +6,7 @@ BEGIN {
     print "var keywords = map[string]int{"
 
     # This variable will be associated with a pipe for intermediate output.
-    sort = "sort"
+    sort = "sort -t \"\\\"\" -k 2,2"
```

Comment:
`sort = "env LC_ALL=C sort"` works. But I think that it is better to sort lines by keywords such as `CHAR` instead of whole lines such as `"CHAR":              CHAR,`. Sorting by the whole lines works because `"` has a less ASCII code that letters. 

So I think that a fix should include `sort = "env LC_ALL=C sort"` and sort lines by keywords.